data/repos.yml

```
@@ -278,6 +278,13 @@
   type: Utilities
   sentry_url: false
 
+- repo_name: govuk-display-screen
+  type: Utilities
+  team: "#govuk-insights-and-analytics"
+  management_url: https://govuk-display-screen-20e334eeb1ba.herokuapp.com/
+  production_hosted_on: heroku
+  description: Webapp to display essential gov.uk content analytics on the office TVs
```

```suggestion
  description: Provides realtime GOV.UK content analytics for digital signage
```

Couple of styling issues in your sentence and think the above might make intent clearer.